draft-ietf-quic-transport.md

```diff
@@ -924,11 +924,14 @@ selected by the client, both to ensure correct routing toward the client and to
 allow the client to validate that the packet is in response to an Initial
 packet.
 
-A zero-length connection ID MAY be used when the connection ID is not needed for
-routing and the address/port tuple of packets is sufficient to identify a
-connection. An endpoint whose peer has selected a zero-length connection ID MUST
-continue to use a zero-length connection ID for the lifetime of the connection
-and MUST NOT send packets from any other local address.
+A zero-length connection ID MAY be used when the connection ID is not needed
+for routing and the destination address and port of incoming packets is
+sufficient to identify a connection (e.g., when there is a single connection
+on a given local port). Note that it is not possible to use the source address
+and port of incoming packets to demultiplex them across connections because a
+peer might multiplex multiple connections on a single address and port and rely
+on its connection IDs for demultiplexing, and the peer's connection IDs are not
+transmitted in the short header packets they send.
```

Comment:
Suggestion: A zero-length connection ID can be used when a connection ID is not needed to correctly route to the correct endpoint.  An endpoint MUST NOT use a zero-length connection ID unless it can use only its IP address and port to identify a connection.  The IP address and port used by a peer cannot be used for routing as these values can change or be used for additional connections.

Reply:
Applied the suggestion with minor changes.